### ISSUE TITLE
Wrong warning about Kernel exec (?)

### ISSUE THREAD

**Issue by PragTob** [jruby/jruby-lint#11]
On Shoes4 we get the following error:

```
./lib/shoes/cli.rb:102: [exec, warning] Kernel#exec does not replace the running JRuby process and may behave unexpectedly
```

however according to the jruby 1.7.0 release notes this should be ok or do I misunderstand it:

```
Kernel#exec now does a true native exec on all platforms
```
